### PR TITLE
Added password strength

### PR DESCRIPTION
**Pull request by robimaliqi**

### CODE REVIEW

File: frontend/src/components/Pages/signUp/SignUp.tsx [hunk 1]
```diff
@@ -1,102 +1,146 @@
...
+        },
+        body: JSON.stringify({ email, password }),
+      });
+      if (response.ok) {
+        // Assuming the signup is successful, relocate the user to the home page
+        navigate("/");
+      } else {
+        // Handle signup failure, such as displaying an error message to the user
+        console.log("Sign up failed");
+        console.log(email);
```
Comment (ValeSer) [inline frontend/src/components/Pages/signUp/SignUp.tsx:84]: this can be removed

File: frontend/src/components/Pages/signUp/SignUp.tsx [hunk 2]
```diff
@@ -1,102 +1,146 @@
...
+        console.log(password);
```
Comment (ValeSer) [inline frontend/src/components/Pages/signUp/SignUp.tsx:85]: this can also be removed

**Review by ValeSer — APPROVED** [review]
Thank you, it looks nice :) I added 2 suggestions :)

**Comment by bjentwistle** [comment]
Robi this is ready to merge please

**Pull request merged**